### PR TITLE
EIP-1: formatting fixes and clarifications

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 1 is in state Active, not Draft or Last Call
 - EIP 2 is in state Final, not Draft or Last Call
 - File README.md is not an EIP
 - File eip-template.md is not an EIP
 - EIP 1812 requires approval from one of (@pelle)

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -94,9 +94,10 @@ Each status change is requested by the EIP author and reviewed by the EIP editor
 Other exceptional statuses include:
 
 * **Abandoned** -- This EIP is no longer pursued by the original authors or it may not be a (technically) preferred option anymore.
-* **Rejected** -- An EIP that is fundamentally broken or a Core EIP that was rejected by the Core Devs and will not be implemented.
-* **Active** -- This is similar to Final, but denotes an EIP which may be updated without changing its EIP number.
-* **Superseded** -- An EIP which was previously final but is no longer considered state-of-the-art. Another EIP will be in Final status and reference the Superseded EIP.
+  * :arrow_right: Draft -- Authors or new champions wishing to pursue this EIP can ask for changing it to Draft status.
+* **Rejected** -- An EIP that is fundamentally broken or a Core EIP that was rejected by the Core Devs and will not be implemented. An EIP cannot move on from this state.
+* **Active** -- This is similar to Final, but denotes an EIP which may be updated without changing its EIP number. An EIP cannot move on from this state.
```
Comment (Arachnid) [inline EIPS/eip-1.md]: It'd be good to clarify that this is just for 'special' EIPs like EIP1.

Comment (axic) [reply EIPS/eip-1.md]: #2196 clarifies those parts.

**Comment by axic** [comment]
@nicksavers @Arachnid any objections merging this?

**Comment by axic** [comment]
@Arachnid @nicksavers @Souptacular @gcolvin can this be merged?

File: EIPS/eip-1.md [hunk 2]
```diff
@@ -71,13 +71,13 @@ These calls generally result in a "rough consensus" around what EIPs should be i
 Following is the process that a successful EIP will move along:
 
 ```
-[ WIP ] -> [ DRAFT ] -> [ LAST CALL ] -> [ ACCEPTED ] -> [ FINAL ]
+[ IDEA ] -> [ DRAFT ] -> [ LAST CALL ] -> [ ACCEPTED ] -> [ FINAL ]
```
Comment (nicksavers) [inline EIPS/eip-1.md]: If we're going to change this here, it should be changed in other places too.

**Comment by axic** [comment]
Merging as nobody was against this, one editor approved and it is mostly clarifications.

**Pull request merged**